Neos.Neos/Resources/Private/Fusion/Prototypes/ImageTag.fusion

```diff
@@ -0,0 +1,37 @@
+# ImageTag object works exactly the same way as image ViewHelper in the Neos.Media package
+prototype(Neos.Neos:ImageTag) < prototype(Neos.Fusion:Tag) {
+  asset = 'pass-the-media-asset'
+  maximumWidth = 2560
+  width = NULL
+  maximumHeight = 2560
+  height = NULL
+  allowCropping = false
+  allowUpScaling = false
+  async = false
+  quality = NULL
+  preset = NULL
+  @context.thumbnail = ${Neos.Media.Image.createThumbnail(
```

Comment:
is that correct that Neos.Neos has a fusion prototype with a Neos.Media dependency?

Why not have the helper in Neos or the fusion prototype in Neos.Media?

im just wondering.

Reply:
Media has no dependency on Fusion but already eel helpers for assets. I would keep it like that.